py3status/modules/xsel.py

```diff
@@ -32,13 +38,25 @@ class Py3status:
     """
     # available configuration parameters
     cache_timeout = 0.5
-    command = 'xsel -o'
+    command = 'xsel -output'
+    click_command = 'xsel --clear --clipboard'
     format = '{selection}'
     max_size = 15
     symmetric = True
+    log_file = None
+
+    selection_cache = None
```

Comment:
This should go under `post_config_hook`. See `twitch_streaming` (broken module) for an example.

Also, we could add `self.log_file = os.path.expanduser(self.log_file)` too to deal with `~/xsel.log` or `~/.config/py3status/xsel.log`.

Reply:
Great idea!